content/en/database_monitoring/guide/connect_dbm_and_apm.md

```diff
@@ -145,6 +146,53 @@ client.query("SELECT 1;")
 
 {{% /tab %}}
 
+{{% tab "Python" %}}
+
+Update your app dependencies to include [dd-trace-py>=1.7.0][1]:
+```
+pip install "ddtrace>=1.7.0"
+```
+
+Install [psycopg2][2] (Note - DBM-APM linking is not currently supported for mysql clients):
+```
+pip install psycopg2
+```
+
+Enable the database monitoring propagation feature by setting the following environment variable:
+   - `DD_TRACE_SQL_COMMENT_INJECTION_MODE=full`
+
+For the best user experience ensure the following environment variables are set in your application:
+   - `DD_SERVICE=(application name)`
+   - `DD_ENV=(application environment)`
+   - `DD_VERSION=(application version)`
+
+Full example:
+```python
+
+import psycopg2
+
+#TODO: update postgres configurations
```

Comment:
Is the TODO meant for the end-user copying that example?

Reply:
Yes